cmake/modules/SwiftModuleInstallation.cmake

```diff
@@ -71,32 +22,20 @@ function(_swift_testing_install_target module)
     set(module_name ${module})
   endif()
 
-  if(NOT SwiftTesting_MODULE_TRIPLE)
-    set(module_triple_command "${CMAKE_Swift_COMPILER}" -print-target-info)
-    if(CMAKE_Swift_COMPILER_TARGET)
-      list(APPEND module_triple_command -target ${CMAKE_Swift_COMPILER_TARGET})
-    endif()
-    execute_process(COMMAND ${module_triple_command} OUTPUT_VARIABLE target_info_json)
-    string(JSON module_triple GET "${target_info_json}" "target" "moduleTriple")
-    set(SwiftTesting_MODULE_TRIPLE "${module_triple}" CACHE STRING "swift module triple used for installed swiftmodule and swiftinterface files")
-    mark_as_advanced(SwiftTesting_MODULE_TRIPLE)
-  endif()
-
-  set(module_dir "${lib_destination_dir}/${module_name}.swiftmodule")
   install(FILES $<TARGET_PROPERTY:${module},Swift_MODULE_DIRECTORY>/${module_name}.swiftdoc
-    DESTINATION "${module_dir}"
+    DESTINATION "${SwiftTesting_INSTALL_SWIFTMODULEDIR}/${module_name}.swiftmodule"
```

Comment:
Is there a strong reason to get rid of `module_dir`? This destination string seems a bit repetitive now, and I wonder if `module_dir` could be preserved but updated to use the new variables

Reply:
Most/many of the projects have more than one module being installed. Creating a local variable for the single module is fine, but I don't think that it is particularly helpful.